adding additional content for welcome page.

carolinambravo opened the pull request:
revised navbar and started formatting welcome page. minor change to text styles.

carolinambravo committed:
adding additional content for welcome page. revised navbar and started formatting welcome page. minor change to text styles.

Cindywangtelanoff reviewed (commented):
Hi Carolina, this is good. However, I saw on my screen that Carolina wants to merge 1 commit into development-carolina from development - I thought it hsould be the other way round?

LfifteenA committed:
Added Heroku Delpoyment Information

LfifteenA committed:
Heroku Deployment Information - Added spacing

LfifteenA committed:
Heroku Deployment Information - Adding Details

LfifteenA committed:
Heroku Deployment Information - Clarified

LfifteenA committed:
Cleaned up Heroku Deployment Information

LfifteenA committed:
Cleaned up Heroku Deployment Information

LfifteenA committed:
Cleaned up Heroku Deployment Info

LfifteenA committed:
Heroku Deployment Information:

tinuola commented:
@carolinambravo are you trying to update your branch or are you trying to update development? See the heading: "carolinambravo  wants to merge 8 commits into development-carolina from development"

LfifteenA committed:
Adding Procfile - Needed for Heroku Deployment

https://devcenter.heroku.com/articles/getting-started-with-nodejs#define-a-procfile

Use a Procfile, a text file in the root directory of your application, to explicitly declare what command should be executed to start your app.

The Procfile in the example app you deployed looks like this:
web: node index.js
This declares a single process type, web, and the command needed to run it. The name web is important here. It declares that this process type will be attached to the HTTP routing stack of Heroku, and receive web traffic when deployed.
Procfiles can contain additional process types. For example, you might declare one for a background worker process that processes items off of a queue.

carolinambravo commented:
Thanks @tinuola, I was trying to merge into development from my branch. I'm going to delete my prior branch, I think all updates have now been made. I just had some git breakthroughs (for now) so hopefully my last round of changes go through correctly.
